packages/react/slider/src/Slider.tsx

```diff
@@ -676,7 +679,7 @@ const BubbleInput = (props: React.ComponentProps<'input'>) => {
    * Adding the `value` will cause React to consider the programatic
    * dispatch a duplicate and it will get swallowed.
    */
-  return <input style={{ display: 'none' }} {...inputProps} ref={ref} />;
+  return <input style={{ display: 'none' }} {...inputProps} ref={ref} defaultValue={value} />;
```

Comment:
Added `defaultValue` here to fix this issue:


https://user-images.githubusercontent.com/175330/120328686-fc867600-c2e2-11eb-8c39-db8b109ac472.mp4

The form change event was firing but these inputs had no initial value so our `legend` gets updated with an undefined value.